spec.html

```diff
@@ -480,7 +479,8 @@ <h1>InnerModuleEvaluation( _module_, _stack_, _index_ )</h1>
               1. Set _module_.[[DFSAncestorIndex]] to min(_module_.[[DFSAncestorIndex]], _requiredModule_.[[DFSAncestorIndex]]).
             1. <ins>Otherwise, set _requiredModule_ to GetCycleRoot(_requiredModule_).</ins>
             1. <ins>If _requiredModule_.[[EvaluationError]] is not *undefined*, return _module_.[[EvaluationError]].</ins>
-            1. <ins>If _cycle_ is *false* and _requiredModule_.[[Status]] is not `"evaluated"`,</ins>
+            1. <ins>Let _cycle_ be *true* if _module_.[[DFSIndex]], _requiredModule_.[[DFSIndex]] and _requiredModule_.[[DFSAncestorIndex]] are all equal.</ins>
```

Comment:
If _requiredModule_ was handled by a separate `InnerModuleEvaluation` call, then the DFSIndex will be a separate numbering operation, so we cannot be comparing indexes across separate numberings. In both the `"evaluated"` and `"evaluating-async"` states for a `requiredModule` this will be the case.